mobify/scooch
# Add a Right to Left option

Add an option for RTL cases
  
**Status**: *Opened for visibility*
 
**Reviewers**: @scalvert @donnielrt @ericmuyser @tedtate @kpeatt 
**Linked PRs**: https://github.com/mobify/scooch/pull/26
  
## Changes
- Added a RTL flag option: `rightToLeft` (defaults to false) for sites that use a right-to-left layout (e.g. eXtra Stores)
- When enabled, this option reverses how Scooch determines next and previous slide movements on dragging interactions
- Currently on eXtra Stores' build, grunt-css-flip handles the RTL inverting of the Scooch controls. We could possibly extend `rightToLeft` flag's functionality to perform these control changes Scooch-side. 

## How to test-drive this PR
- Pull down this change
- Init a new Scooch with `rightToLeft` option set to true: `$scooch.scooch({rightToLeft: true});`
- Check whether dragging in a left-to-right manner advances Scooch, and returns to previous slide on a right-to-left dragging interaction

## Note
- These changes are stacked on top of @ericmuyser 's fix for logic to auto-hide arrows
